lib/omnibus/packagers/deb.rb

```diff
@@ -390,6 +415,73 @@ def compression_params
     end
 
     #
+    # Check for the existance of a command in +ENV['PATH']+. Exit code +127+
+    # tells us the shell could not find the program.
+    #
+    # @return [bool]
+    #
+    def check_command_exists(command)
```

Comment:
This probably belongs in the `Util` class

Reply:
I believe, that it actually not need to be provided at all, as there is `which` method provided and used already in various places, as per: https://github.com/chef/omnibus/blob/master/lib/omnibus.rb#L170; and we should see whether it could be used here instead of `chek_command_exists`.

Reply:
I'd agree with @kwilczynski here. I can likely refactor the reliant code to make use of the existing `which`, as this essentially performs the same tasks, but only returns boolean vs a string/nil.